app/src/main/java/org/literacyapp/battery/BatteryReceiver.java

```diff
@@ -19,7 +19,7 @@ public void onReceive(Context context, Intent intent) {
 
         Log.d("Battery status: ", String.valueOf(batteryPct));
 
-        if(batteryPct <= 43 || batteryPct != 1.0) {
+        if(batteryPct <= 10 || batteryPct != 1.0) {
```

Comment:
It checks 1.0 because when the device is connected battery status changes to that value.